From 9ee7047a312e40d6f927027178e83737bd69f5a6 Mon Sep 17 00:00:00 2001
From: Alain Volmat <alain.volmat@foss.st.com>
Date: Wed, 27 Mar 2024 15:07:45 +0100
Subject: [PATCH 1064/1141] dt-bindings: media: fix bindings for dcmipp driver

Fix dt_binding_check error related to binding wording in the
title and missing description of the pclk-max-frequency port
property.

Change-Id: Ic1df631cdffde92876cd9942653a58637e67c960
Signed-off-by: Alain Volmat <alain.volmat@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/369706
Reviewed-by: Hugues FRUCHET <hugues.fruchet@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Philippe CORNU <philippe.cornu@foss.st.com>
Reviewed-by: Philippe CORNU <philippe.cornu@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 .../devicetree/bindings/media/st,stm32-dcmipp.yaml          | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/Documentation/devicetree/bindings/media/st,stm32-dcmipp.yaml b/Documentation/devicetree/bindings/media/st,stm32-dcmipp.yaml
index 5c80eb28489d..556a3d2e7f89 100644
--- a/Documentation/devicetree/bindings/media/st,stm32-dcmipp.yaml
+++ b/Documentation/devicetree/bindings/media/st,stm32-dcmipp.yaml
@@ -4,7 +4,7 @@
 $id: http://devicetree.org/schemas/media/st,stm32-dcmipp.yaml#
 $schema: http://devicetree.org/meta-schemas/core.yaml#
 
-title: STMicroelectronics STM32 DCMIPP Digital Camera Memory Interface Pixel Processor binding
+title: STMicroelectronics STM32 DCMIPP Digital Camera Memory Interface Pixel Processor
 
 maintainers:
   - Hugues Fruchet <hugues.fruchet@foss.st.com>
@@ -59,6 +59,10 @@ properties:
           hsync-active: true
           vsync-active: true
 
+          pclk-max-frequency:
+            description:
+              Maximum acceptable pixel clock frequency.
+
         required:
           - pclk-sample
 
-- 
2.39.2

